script/refresh_sitemap

```diff
@@ -211,9 +209,11 @@ def map_one_object_type(params, comments, last_update)
 end
 
 def update_times_of_new_objects(times, table, last_update)
-  Name.connection.select_rows(%(
-    SELECT id, updated_at FROM #{table} WHERE updated_at > "#{last_update}"
-  )).each do |id, time|
+  table = Arel::Table.new(table.to_sym)
+  Name.connection.select_rows(
+    table.where(table[:updated_at] > last_update).
+          project(table[:id], table[:updated_at]).to_sql
+  ).each do |id, time|    
```

Comment:
Would you mind explaining why pluck won't work here?  Oh, is it because table is dynamic?

Reply:
Oh hey!  Would it feel better to change the callers of update_times_of_new_objects to supply the model class instead of table name?  Then maybe you could use pluck?  With such a distinctive method name, I bet it would be easy to track down all the callers.  (I bet there aren't many.)

Reply:
Because table is dynamic, yes. You can't do table.pluck on an arel table, only on a model.

Reply:
That sounds good! Lemme see if i can make it work.

Reply:
Done